tests/utils.py

```diff
@@ -1,10 +1,18 @@
+import asyncio
+import time
 import uuid
 
 
 def unique_test_cache_name() -> str:
     return f"python-test-{uuid_str()}"
 
 
+def unique_test_vector_index_name() -> str:
+    name = f"python-test-{uuid_str()}"
+    # TODO remove this
+    return name.replace("-", "_")
```

Comment:
```suggestion
    return f"python-test-{uuid_str()}"
```

We can remove that now :)